index.html

```diff
@@ -126,23 +121,20 @@ <h3 class="work-role h3-common-styles">Back End Dev</h3>
                   <button type="button" class="works-btn">javascript</button>
                 </li>
               </ul>
-              <button class="see-project-btn" type="button">See Project</button>
+              <button class="see-project-btn seeWork1Btn" type="button">
+                See Project
+              </button>
             </div>
           </div>
         </div>
 
         <!-- ---------------Work 2----------------- -->
         <div class="grid-item">
           <div class="card-container">
```

Comment:
- [ ] You have made brilliant efforts here to optimize your page :wink: but I think that this element i.e. `card-container` should be empty, especially because you have an array of your projects where you have implemented some logic expected to load your project when the page loads. Having these card details in you index.html again is an unnecessary duplication that may affect the performance of your site, and truly loading from your `script.js` is a brave move - it reduces load time, and increase the efficient uses of visitor's resources. As requested by the previous reviewer, kindly remove the card details.

Reply:
@AbayomiOlaoye Thanks for the feedback. However, This card does not wrap the popup window. It wraps the cards as requested in the previous projects. I do not understand. Could you please try explaining this issue

Reply:
Okay @Amen-Musingarimi , can you spare me a minute please?

Reply:
@AbayomiOlaoye yes please

Reply:
- The `card-container` should be empty now that you have the content loading from `script.js` - `div class="card-container"></div>`. You can then make this your point of entry by manipulating the DOM to populate on page load.